.gitlab-ci.yml

```diff
@@ -62,12 +62,15 @@ build:ts:
     - export REACT_APP_API_BASE_URL=$ASAP_API_URL
     - export REACT_APP_ALGOLIA_INDEX=$ALGOLIA_INDEX
     - export REACT_APP_ALGOLIA_APP_ID=$(aws ssm get-parameter --name "algolia-app-id-dev" --query Parameter.Value --output text)
+    - yarn install --immutable --immutable-cache
```

Comment:
just like we said i think its best to move all this into a different job that uses this build artifact, in there you can run yarn install to get the binary you need but i dont think yarn install should be run in the CI before bundling